app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  validates :name, presence: true, length: { maximum: 50 }
+
+  validates :name, uniqueness: true
+
+  has_many :post
+
+  has_many :comment
```

Comment:
These relationships are wrong. Kindly check below :arrow_down: 

- [ ] A user has many posts not post.

- [ ] A user has many comments not comment

### hint
Always remember when we are talking about has_many the model you are referring to should always be pluralized

![posts](https://user-images.githubusercontent.com/32361963/98927031-14361280-24e9-11eb-9c56-2e3b5dea650e.png)